analyzers/tests/SonarAnalyzer.UnitTest/TestCases/Hotspots/CookieShouldBeHttpOnly_Net.cs

```diff
@@ -20,7 +20,7 @@ class Program
 
     void InitializerSetsNotAllowedValue()
     {
-        CookieOptions c1 = new () { HttpOnly = false }; // FN
-        CookieOptions c2 = new () { Secure = true }; // FN, as HttpOnly is not set
+        CookieOptions c1 = new () { HttpOnly = false }; // Noncompliant
+        CookieOptions c2 = new () { Secure = true }; // Noncompliant
```

Comment:
_Nitpick (for next time):_ I'd align these comments when I'm touching old code (only these two, not those above)